app/components/IndicatorChart/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { ContainerCard } from '@the-deep/deep-ui';
+import { _cs } from '@togglecorp/fujs';
+import {
+    Line,
+    LineChart,
+    ResponsiveContainer,
+    XAxis,
+    YAxis,
+    Tooltip,
+    Legend,
+} from 'recharts';
+
+import styles from './styles.css';
+
+type IndicatorOverviewType = {
+    id: number;
+    month: string;
+    percentage: number;
+};
+interface IndicatorChartProps {
+    className?: string;
+    heading?: React.ReactNode;
+    headerDescription?: React.ReactNode;
+    headingSize?: 'large' | 'small' | 'extraSmall' | 'medium' | 'extraLarge' | undefined;
```

Comment:
If you want to accept the types given by 'headingSize' from 'ContainerCard', I suggest you use that type instead of creating your own.

Reply:
Done